config/base.go

```diff
@@ -26,8 +28,23 @@ func save(filePath string, cfg interface{}) error {
 	return encoder.Encode(cfg)
 }
 
-// load reads file, parses json and stores in cfg struct
-func load(filePath string, cfg interface{}) error {
+// Load reads file, parses json and stores in cfg struct
+func Load(filePath string, cfg interface{}) error {
```

Comment:
Since `load` is a helper function, I don't think we should expose it to external users directly. Maybe we could move it into an internal package.